docs/database.md

```diff
@@ -0,0 +1,49 @@
+# Database
+
+Software Factory applications that require a database are configured to use Postgresql as a backend.  This service can be provided to a given application in a few supported ways:
+
+## UDS Postgres Operator Package
+
+If your environment requires an in-cluster solution, you can make use of the [UDS Postgres Operator Package](https://github.com/defenseunicorns/uds-package-postgres-operator).  This can be included in your `uds-bundle.yaml` and is configured through the `postgresql` Helm value to create the required CRs for the operator.  Below is an example of what this configuration would look like for an application (GitLab in this case):
+
+```yaml
+postgresql:
+    enabled: true
+    teamId: "uds"
+    volume:
+        size: "10Gi"
+    numberOfInstances: 2
+    users:
+        gitlab.gitlab: []
+    databases:
+        gitlabdb: gitlab.gitlab
+    version: "13"
+    ingress:
+        remoteGenerated: Anywhere
```

Comment:
this will need to be updated to the new array format since that's about to get merged in.